src/patternfly/components/Table/table.scss

```diff
@@ -715,6 +715,8 @@
 .#{$table} .#{$table}__draggable {
   --#{$table}--cell--MinWidth: 0;
   --#{$table}--cell--Width: 1%;
+
+  max-width: none;
```

Comment:
WDYT about using something like fit-content or min-content? I'm not sure it's any different functionally but worries me less 🙃

Reply:
Works for me! @srambach updated

Reply:
Looks like "fit-content" doesn't work in FF, though "none" does

Reply:
👍🏻